puppet/modules/web/manifests/init.pp

```diff
@@ -84,10 +84,24 @@
   create_resources(web::htpasswd, $htpasswds)
 
   # YUM
+  $yum_directory = '/var/www/vhosts/yum/htdocs'
+  $yum_directory_config = [
+    {
+      path    => $yum_directory,
+      options => ['Indexes', 'FollowSymLinks', 'MultiViews'],
+    },
+    {
+      path     => '.+\.(bz2|gz|rpm)$',
```

Comment:
Do we have any `.xz` files?

Reply:
I didn't `find(1)` any, but can still be added for future-proofing if you want?